### DISCUSSION TITLE
Format for HTTP requests?

### DISCUSSION THREAD

**Discussion by proteus-dev** [DeabLabs/cannoli#12] [Q&A] [answered]
Hello again,

I am a noob when it comes to coding... what is the correct string format for a cannoli to trigger an HTTP request? 
Can I just curl as below?

curl 'https://<PROJECT_REF>.supabase.co/rest/v1/todos' \
-H "apikey: <ANON_KEY>" \
-H "Authorization: Bearer <ANON_KEY>"

**Comment by blindmansion** [comment, marked as answer]
Check out the action node section of the college, there's an example of how it's set up. It's going to be a JSON object with all the same fields that are in the action mode template on the settings page.

**Reply by proteus-dev** [reply]
ah thanks! was hidden in plain sight for me... I reviewed all college articles but my brain parsed this node's content "just as some JSON"... Thanks for showing me the direction!

<img width="1088" alt="image" src="https://github.com/DeabLabs/cannoli/assets/121372604/805a6f09-0099-476d-b0f3-bd3d50057295">

**Reply by ttodosi** [reply]
Anyone sent 10,000 emails yet?

**Reply by blindmansion** [reply]
lol I don't think so, but maybe. There would have to be either a loop group with 1000 loops or 1000 nodes of you wanted to do it in parallel

But forEach groups are in the works, and that would make this much easier
